ghost.env.enc

```diff
@@ -0,0 +1,3 @@
+��e��[���CTL{����a��/�p�R��Dt#��e4l��=�e�Ba��?�MH[[��E�#�d�J5�qX��0B��zXzH���b�6�U�xBD_���RW�ЪP]Q|��9��&��֛R���ՙ�t�b�>O�rt��dr�I�����G�sv�6�6NHp;����P7%�L�3�S3
+U&�6�ġ)UV|}[E��U��.MIR,>�u��q������W��zߍ�Bt2�6�?���~��C�=���;���5Y���'d߱^���N�Wef}�?@$#�P�gT���:|�ș�{�U��K|S'��l��c�>��\��#���/��،
+�A���|��\���ԶL�\�f����%����s���̳�K�d鉚�d�A��RkB�w�8�{���x�0ذB
���<'�Á�WV��S�� �nOV@���H��<!��Z���xl�<�G�<�8���}
```

Comment:
Does this file need to be marked "binary" -- like the next one?

Reply:
yep